engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/runtime/task/TaskQueryDto.ftl

```diff
@@ -0,0 +1,655 @@
+                `gt` - greater than;
+                `gteq` - greater than or equal to;
+                `lt` - lower than;
+                `lteq` - lower than or equal to;
+                `like`.
+                `key` and `value` may not contain underscore or comma characters." />
+  
+    <@lib.property
+        name = "caseInstanceVariables"
+        type = "array"
+        dto = "VariableQueryParameterDto"
+        desc = "A JSON array to only include tasks that belong to a case instance with variables
+                with certain values. The array consists of JSON objects with three properties
+                `name`, `operator` and `value`. `name` is the variable name, `operator` is the
+                comparison operator to be used and `value` the variable value. `value` may be of
+                type `String`, `Number` or `Boolean`.
+  
+                Valid `operator` values are:
+                `eq` - equal to;
+                `neq` - not equal to;
+                `gt` - greater than;
+                `gteq` - greater than or equal to;
+                `lt` - lower than;
+                `lteq` - lower than or equal to;
+                `like`.
+                `key` and `value` may not contain underscore or comma characters." />
+  
+    <@lib.property
+        name = "variableNamesIgnoreCase"
+        type = "boolean"
+        defaultValue = "false"
+        desc = "Match all variable names in this query case-insensitively. If set
+                `variableName` and `variablename` are treated as equal." />
+  
+    <@lib.property
+        name = "variableValuesIgnoreCase"
+        type = "boolean"
+        defaultValue = "false"
+        desc = "Match all variable values in this query case-insensitively. If set
+                `variableValue` and `variablevalue` are treated as equal." />
+  
+    <@lib.property
+        name = "parentTaskId"
+        type = "string"
+        desc = "Restrict query to all tasks that are sub tasks of the given task. Takes a task id." />
+
+    <@lib.property
+        name = "orQueries"
+        type = "array"
+        dto = "ProcessInstanceQueryDto"
+        desc = "A JSON array of nested process instance queries with OR semantics.
+                A process instance matches a nested query if it fulfills at least one of the query's predicates.
+                With multiple nested queries, a process instance must fulfill at least one predicate of each query (Conjunctive Normal Form).
+                All process instance query properties can be used except for: `sorting`
+                See the [User guide](${docsUrl}/user-guide/process-engine/process-engine-api/#or-queries) for more information about OR queries." />
+
+    "sorting": {
+      "type": "array",
+      "description": "Apply sorting of the result",
+      "items":
+
+        <#assign last = true >
+        <#assign sortByValues = ['"instanceId"', '"definitionId"', '"definitionKey"', '"businessKey"', '"tenantId"']>
```

Comment:
`sortByValues` are wrong:
instanceId, caseInstanceId, dueDate, executionId, caseExecutionId,assignee, created, description, id, name, nameCaseInsensitive, priority, processVariable, executionVariable, taskVariable, caseExecutionVariable, and caseInstanceVariable